frontend/src/components/Login.tsx

```diff
@@ -1,9 +1,80 @@
-import React from 'react'
+import React, {useState} from 'react'
+
 
 function Login() {
-  return (
-    <div className='p-6'>Login</div>
-  )
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [isEmailValid, setIsEmailValid] = useState(true);
+    const [isPasswordValid, setIsPasswordValid] = useState(true);
+    const [submitted, setSubmitted] = useState(false);
+
+    const handeEmailChange = (event: React.FormEvent<HTMLInputElement>) => {
+        setEmail(event.currentTarget.value);
+        setIsEmailValid(true);
+        setSubmitted(false);
+    }
+
+    const handePasswordChange = (event: React.FormEvent<HTMLInputElement>) => {
+        setPassword(event.currentTarget.value);
+        setIsPasswordValid(true);
+        setSubmitted(false);
+    }
+
+    const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        if (email != "test@email.com") {
+            setIsEmailValid(false);
+            setSubmitted(false);
+        }
+        if( password != "password") {
+            setIsPasswordValid(false);
+            setSubmitted(false);
+        }
+        if (isPasswordValid && isEmailValid) {
+            setSubmitted(true);
+        }
+    };
+
+    return (
+        <div className="flex flex-col items-center pt-20">
+            <div className="w-2/5 pb-14 text-center text-2xl font-medium">
+                <p>Logowanie</p>
+            </div>
+            <div className='w-96 font-medium'>
+                <form className=" " onSubmit={handleSubmit} method={"POST"}>
+                    <label className="flex flex-col py-2">
+                        <span className="flex py-2 after:content-['*'] after:text-red-600 font-bold">Email</span>
+                        {!isEmailValid ?
+                            <input type="email" name="email" placeholder={"adres@email"} value={email} onChange={handeEmailChange}
```

Comment:
there are other implemented ways to tell user that it's not correct email, you can get rid of ternary operator and leave only one input